test/format/failure_configs/CONFIG.WT-9807

```diff
@@ -0,0 +1,102 @@
+############################################
```

Comment:
Please hold on for this config - Let's try to find one that reproduces the issue on `develop`.

Reply:
All good, I could confirm.